apps/myaccount/src/components/change-password/change-password.tsx

```diff
@@ -80,6 +81,67 @@ export const ChangePassword: FunctionComponent<ChangePasswordProps> = (props: Ch
 
     const endUserSession = useEndUserSession();
 
+    interface iconProps {
+        class : string;
```

Comment:
Avoid using `class` or any keywords in attribute namespaces.

```suggestion
        cssClassName : string;
```